### PR TITLE
Fixed quest drop modal

### PR DESCRIPTION
**Pull request by TheHollidayInn**
#7359
### Changes

---

UUID:

### CODE REVIEW

**Comment by paglias** [comment]
Some client side tests are failing

**Comment by TheHollidayInn** [comment]
Yea, sorry. Updating soon.

**Comment by TheHollidayInn** [comment]
Looks like server side errors now. This passes locally, so not sure that my PR is affecting that. Rebasing and pushing though.

File: test/spec/controllers/partyCtrlSpec.js [hunk 1]
```diff
@@ -51,7 +51,9 @@ describe("Party Controller", function() {
         var syncParty = sinon.stub(groups.Group, 'syncParty')
         syncParty.returns(Promise.resolve(groupResponse));
         $controller('PartyCtrl', { $scope: scope, $state: state, User: User });
-        expect(state.is).to.be.calledOnce; // ensure initialization worked as desired
+        // @TODO: I have update the party ctrl to sync the user whenever it is called rather than only on the party page
```
Comment (crookedneighbor) [inline test/spec/controllers/partyCtrlSpec.js:54]: I don't understand this todo comment.

Comment (TheHollidayInn) [inline test/spec/controllers/partyCtrlSpec.js:54]: I removed the syncing party when the state is the state specified in the test. We don't have complete yet coverage, so if the dramatic change starts breaking a bunch of stuff, I left this reminder for me to follow up. I can just remove it, though.

**Pull request merged**